### PR TITLE
[Mobile] Update SASS transform config

### PR DESCRIPTION
**Pull request by geriux**

### CODE REVIEW

File: packages/react-native-editor/package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
 		"start": "echo \"\\x1b[33mThe start command is not available in this project. It is strongly recommended to use \\x1b[1:33mstart:reset\\x1b[0m\\x1b[33m to perform some cleanup when starting the metro bundler.\nOr you may use \\x1b[1:33mstart:quick\\x1b[0m\\x1b[33m for a quicker startup, but this may lead to unexpected javascript errors when running the app.\\x1b[0m\"",
 		"start:debug": "node --inspect-brk node_modules/.bin/react-native start",
 		"start:reset": "npm run clean:runtime && npm run start:quick -- --reset-cache",
-		"start:quick": "react-native start",
+		"start:quick": "react-native start --experimental-debugger",
```
Comment (dcalhoun) [inline packages/react-native-editor/package.json:83]: Do you think we should enable this globally or let contributors pass this as desired via `npm run start:quick -- --experimental-debugger`?

File: packages/react-native-editor/sass-transformer.js [hunk 2]
```diff
@@ -117,7 +122,12 @@ function transform( src, filename, options ) {
 						path.resolve( path.dirname( filename ), urlPath.dir )
 					); // Add the file's dir to the search array.
 				}
-				const f = findVariant( urlPath.name, exts, incPaths );
+				const f = findVariant(
+					urlPath.name,
+					exts,
+					incPaths,
+					options.projectRoot
```
Comment (dcalhoun) [inline packages/react-native-editor/sass-transformer.js:129]: How does `projectRoot` arrive here? Is it automatically provided from the Metro config?

**Review by dcalhoun — APPROVED** [review]
The Metro server successfully starts with this, so I think we should merge this and iterate on it. Thank you!

File: packages/react-native-editor/package.json (hunk 1 above)
Comment (geriux) [reply packages/react-native-editor/package.json:83]: Good question, I think it's handy to have it and you just decide if you want to open it or not.

File: packages/react-native-editor/sass-transformer.js (hunk 2 above)
Comment (geriux) [reply packages/react-native-editor/sass-transformer.js:129]: For what I could see yeah even if you don't manually set a `projectRoot` it'll use the value React Native detects.

**Pull request merged**